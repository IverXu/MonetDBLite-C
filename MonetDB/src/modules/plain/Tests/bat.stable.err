stderr of test 'bat` in directory 'src/modules/plain` itself:

#> Mtest:	other bat
#> Mtest:	./bat bat
#> Mlog:	Mtimeout -timeout 600 Mserver -yaccsilent -d8 -db modules_plain bat.mil
