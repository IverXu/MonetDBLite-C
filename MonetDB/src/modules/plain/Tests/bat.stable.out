stdout of test 'bat` in directory 'src/modules/plain` itself:

#> Mtest:	other bat
#> Mtest:	Mtimeout -timeout 420 ./bat bat
#> Mlog:	Mtimeout -timeout 210 Mserver -modulesilent -yaccsilent -d8 -db modules_plain bat.mil
# Monet Database Server V4.1
# Copyright (c) 1993-1999, CWI & SION. All rights reserved.
# compiled for IRIX646.5/32bit on Fri Jan 14 01:04:33 MET 2000.
#script:bat.mil

loaded;
#---------------------------------#
# module	| usage_count	  #
#---------------------------------#
[ "algebra",	  1		  ]
[ "arith",	  1		  ]
[ "bat",	  1		  ]
[ "kernel",	  1		  ]
[ "str",	  1		  ]
[ "sys",	  1		  ]
[ "trans",	  1		  ]

Try: modules;                - for other modules.
     ls;                     - for persistent bat listing.
     sigs("<module>")        - for available commands per module.
     help("<command>")       - for help one-liner for a command.

#You are system administrator (adm)!
testje := new(int,str);

testje.insert(1,"one");
testje.insert(2,"two");
testje.insert(3,"three");
testje.insert(1,"een");
testje.insert(2,"twee");
testje.insert(3,"drie");

printf("* Example BAT\n");
* Example BAT
testje.print;
#-------------------------#
# BAT:	  tmp_25	  #
# (int)	  (str)		  #
#-------------------------#
[ 1,	  "one"		  ]
[ 2,	  "two"		  ]
[ 3,	  "three"	  ]
[ 1,	  "een"		  ]
[ 2,	  "twee"	  ]
[ 3,	  "drie"	  ]

deltest := new(int,str);
deltest.insert(1,"one");

printf("\n* deleteBuns: Delete the exact BUN\n");

* deleteBuns: Delete the exact BUN
deltest.print;
#-----------------#
# BAT:	  tmp_26  #
# (int)	  (str)	  #
#-----------------#
[ 1,	  "one"	  ]
testje.deleteBuns(deltest);
testje.print;
#-------------------------#
# BAT:	  tmp_25	  #
# (int)	  (str)		  #
#-------------------------#
[ 3,	  "drie"	  ]
[ 2,	  "two"		  ]
[ 3,	  "three"	  ]
[ 1,	  "een"		  ]
[ 2,	  "twee"	  ]

deltest := new(int,int);
deltest.insert(2,2);
deltest.insert(3,3);

printf("\n* delete: Delete the BUNs with corresponding head values\n");

* delete: Delete the BUNs with corresponding head values
deltest.print;
#-----------------#
# BAT:	  tmp_27  #
# (int)	  (int)	  #
#-----------------#
[ 2,	  2	  ]
[ 3,	  3	  ]
testje.delete(deltest);
testje.print;
#-----------------#
# BAT:	  tmp_25  #
# (int)	  (str)	  #
#-----------------#
[ 1,	  "een"	  ]
